src/main/java/seedu/recipe/ui/MainWindow.java

```diff
@@ -142,8 +145,8 @@ void fillInnerParts() {
         cookedListPanel.setChart(goalsCountList);
 
         //using recipe list as stub for achievements, to be edited later todo
```

Comment:
Thanks for the work! I think once the EOF errors are fixed it should be good to merge. Just a suggestion, maybe using TODO comments at the hardcoded parts to navigate through what you have to change later might be helpful. Also, updating the comments such as "using recipe stub" can be removed too once you have adjusted the code so that it won't be confusing.

Reply:
Hi! I think that this would be good to merge as well, once you add a new line in the json files to fix the EOF errors to pass the travis test :)

Reply:
okay guys thanks for ur advices!! the travis CI test works now! :)